app/src/main/java/org/simple/clinic/overdue/callresult/CallResult.kt

```diff
@@ -54,7 +58,9 @@ data class CallResult(
         removeReason = null,
         outcome = Outcome.AgreedToVisit,
         timestamps = Timestamps.create(clock),
-        syncStatus = syncStatus
+        syncStatus = syncStatus,
+        patientId = appointment.patientUuid,
+        facilityId = appointment.facilityUuid
```

Comment:
The appointment facility id is not always the same as the user's current facility. Did we check Prabhanshu to see what we want here?